zebra/router-id.c

```diff
@@ -253,6 +253,34 @@ DEFUN (no_router_id,
 	return CMD_SUCCESS;
 }
 
+DEFUN (show_router_id,
+       show_router_id_cmd,
+       "show router-id [vrf NAME]",
+       "Show the configured router-id\n"
+       VRF_CMD_HELP_STR)
+{
+        int idx_name = 3;
+
+        vrf_id_t vrf_id = VRF_DEFAULT;
```

Comment:
router-id is per protocol, per VRF. We can handle the protocols and non-default VRFs later, but I think printing the VRF name in the output along with router-id will be useful. In the current code it will just print default though.

Reply:
@srimohans, non default vrf are already handled.   
```
show router-id [vrf NAME]
```
Do you want the output to always show vrf  info even for the default case as in 
```
zebra:
   router-id 1.2.3.4 vrf blue
```